core/src/main/scala/akka/persistence/cassandra/healthcheck/AkkaPersistenceCassandraHealthCheck.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.cassandra.healthcheck
+
+import akka.actor.ActorSystem
+import akka.event.Logging
+import akka.pattern.{ ask, AskTimeoutException }
+import akka.persistence.Persistence
+import akka.persistence.cassandra.journal.CassandraJournal.{ HealthCheckQuery, HealthCheckResponse }
+import akka.util.Timeout
+
+import scala.concurrent.{ ExecutionContextExecutor, Future }
+import java.util.concurrent.TimeUnit.MILLISECONDS
+
+import scala.util.control.NonFatal
+
+class AkkaPersistenceCassandraHealthCheck(system: ActorSystem) extends (() => Future[Boolean]) {
+
+  private[akka] val log = Logging.getLogger(system, getClass)
+  private implicit val ec: ExecutionContextExecutor = system.dispatcher
+
+  private val healthCheckSettings = new HealthCheckSettings(system, system.settings.config)
+  private val journalPluginId = s"${healthCheckSettings.pluginLocation}.journal"
+  private val journalRef = Persistence(system).journalFor(journalPluginId)
+
```

Comment:
instead of using `system.dispatcher` we can use
```
private implicit val ec: ExecutionContext = system.dispatchers.lookup(journalPluginId + ".plugin-dispatcher")
```

Reply:
Fixed